DigrammingTool-App/src/test/java/com/diagrammingtool/app/controller/UserRegistrationControllerTest.java

```diff
@@ -40,8 +38,8 @@ class UserRegistrationControllerTest {
 
 //		 assertEquals(HttpStatus.OK, responseEntity.getStatusCode());
 //	     assertEquals(user1, responseEntity.getBody());
-//		 
-//	   }
+		 
+	  // }
 	 
 	@Test
```

Comment:
Assert that the response status is OK and the returned list of users matches the expected list